### PR TITLE
First Week Assignment

### PR DESCRIPTION
**Pull request by 890534**
Suggestions

### CODE REVIEW

File: Selection.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import com.sun.org.apache.bcel.internal.generic.ARETURN;
+
+import java.util.Scanner;
+
+public class Selection {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter Your Choice");
+        String a = scanner.nextLine();
+        if(a.equals("1.Black and White")){
```
Comment (VBhaskarachary) [inline Selection.java]: Pls use proper naming

File: Reverse.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        public static void main(String[] args) {
+            Scanner scanner=new Scanner(System.in);
+            System.out.println("Enter data");
+            String data=scanner.nextLine();
+
+            System.out.println(ReverseAString(data));
+            System.out.println(CountVowels(data));
+        }
+        public static String ReverseAString(String data){
+            String output="";
```
Comment (VBhaskarachary) [inline Reverse.java]: pls dont use static

**Review by VBhaskarachary — APPROVED** [review]
Changes looks good

**Comment by 890534** [comment]
okay Bhaskar.

**Pull request merged**